src/lib/tokenizers/alpheios-remote-tokenizer.js

```diff
@@ -0,0 +1,75 @@
+import { ClientAdapters } from 'alpheios-client-adapters'
+
+export default class AlpheiosRemoteTokenizer {
+  /**
+     * Starting point for tokenize workflow
+     * @param {SourceText} docSource - docSource for tokenize
+     * @param {String} idPrefix - prefix for creating tokens idWord
+     * @returns {[Objects]} - array of token-like objects, would be converted to Tokens outside
+     */
+  static async tokenize (docSource, idPrefix, tokenizeParams) {
+    var textBlob = this.convertStringToBinaryBlob(docSource.text)
+    const fetchOptions = Object.assign({
+      lang: 'lat',
+      textType: 'text',
+      segments: 'singleline'
+    }, tokenizeParams)
+
+    const adapterTokenizerRes = await ClientAdapters.tokenizationGroup.alpheios({
+      method: 'getTokens',
+      params: {
+        text: textBlob,
+        fetchOptions
+      }
+    })
+
+    if (adapterTokenizerRes.errors.length > 0) {
+      adapterTokenizerRes.errors.forEach(error => console.log(error))
+    }
+
+    return {
+      segments: this.formatTokens(adapterTokenizerRes.result.segments, docSource.textType, idPrefix)
+    }
+  }
+
+  /**
+   * Convert to binary to save line breaks for the remote tokenization service
+   * @param {String} text - source text to convert to binary
+   *
+   *  @returns {Blob}
+   */
+  static convertStringToBinaryBlob (text) {
+    const out = []
+
+    for (var i = 0; i < text.length; i++) {
+      out[i] = text.charCodeAt(i)
+    }
+
+    const outBinaryArray = new Uint8Array(out)
+
+    return new Blob([outBinaryArray], {
+      type: 'text/plain'

```

Comment:
question: is this only necessary for plain text input in the textarea? /Does posting of an uploaded file work without this?

Reply:
I am not sure that I understood the question correctly 
We send text from textarea, it is always String - so I need to convert it to binary.

Do you mean here , that we should create a file? a send it as a windows.fetch? But I do the same.
What is your suggestion?

Reply:
Upload allows you to upload a text file, right? That's what I'm referring to

Reply:
Oh, I see - no the source of text - was it typed manually or uploaded from the file does not refered to each other.
This conversion we need to convert source text from textarea to binary file - to send to remote alpheios tokenization service.